src/Components/TreeView/TreeView.js

```diff
@@ -195,13 +187,7 @@ const File = ({
 											className="manage_icon"
 											onClick={() => {
 												onModal('loading');
-												api('GET', `/get/${containerItem.containerId}/${objectItem.address.objectId}`, {}, {
-													[ContentTypeHeader]: "application/json",
-													[AuthorizationHeader]: `Bearer ${walletData.tokens.object.token}`,
-													[BearerOwnerIdHeader]: walletData.account.address,
-													[BearerSignatureHeader]: walletData.tokens.object.signature,
-													[BearerSignatureKeyHeader]: walletData.publicKey,
-												}).then((data) => {
+												api('GET', `/get/${containerItem.containerId}/${objectItem.address.objectId}`).then((data) => {
```

Comment:
But if it's private I will get the same error, right?

Reply:
Yes, `Object is not accessible without a token`